src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -427,9 +428,9 @@ public void search(boolean onlyDataRelatedToMe) {
             }
 
             // populate preview counts: https://redmine.hmdc.harvard.edu/issues/3560
-            previewCountbyType.put("dataverses", 0L);
-            previewCountbyType.put("datasets", 0L);
-            previewCountbyType.put("files", 0L);
+            previewCountbyType.put(BundleUtil.getStringFromBundle("dataverses"), 0L);
```

Comment:
This text isn't actually shown anywhere, correct? I don't think there's a need to localize it.

Reply:
@scolapasta - This shows in the facets 

![image](https://user-images.githubusercontent.com/12862735/56766966-a7c5b800-6778-11e9-9934-07b3e97b9820.png)

Reply:
@JayanthyChengan Sorry, what I meant to say is that the text isn't gotten from this previewCountbyType hashmap, as far as I can tell (in the xhtml, I see calls to "{bundle['dataverse.results.types.dataverses']}" and "{bundle['dataverse.results.types.datasets']}". If that's true, then we don't need to localize the internal keys to hashmaps like this. 

(or am I missing something?)

Reply:
@scolapasta - I replied for this comment to your email.  Please let me know, if I need to forward the email again.  Thanks

Reply:
@JayanthyChengan Can you go ahead and forward the e-mail again? I couldn't find the reference to this specific comment.

Reply:
@scolapasta - just forwarded the e-mail.